libraries/SD_MMC/examples/SDMMC_Test/SDMMC_Test.ino

```diff
@@ -172,8 +210,37 @@ void testFileIO(fs::FS &fs, const char * path){
 
 void setup(){
     Serial.begin(115200);
-    if(!SD_MMC.begin()){
-        Serial.println("Card Mount Failed");
+    while(!Serial) { delay(10); }
+    Serial.println("SDMMC_Test.ino starting!");
+
+    // If you are using any other ESP32-S3 board than ESP32-S3-USB-OTG which has preset default pins, you will
+    // need to specify the pins with the following example of SD_MMC.setPins()
+    // If you want to use only 1-bit mode, you can use the line with only one data pin (d0) begin changed.
+    // Please note that ESP32 does not allow pin changes and will fail unless you enter the same pin config as is the hardwired.
+#if defined(SOC_SDMMC_USE_GPIO_MATRIX) && not defined(BOARD_HAS_SDMMC)
+    //if(! SD_MMC.setPins(clk, cmd, d0)){ // 1-bit line version
+    if(! SD_MMC.setPins(clk, cmd, d0, d1, d2, d3)){ // 4-bit line version
+        Serial.println("Pin change failed!");
+        return;
+    }
+#endif
+
+/*
+// This is waiting for implementation in peripheral manager (periman)
+    if(use_1_bit_mode){
+        Serial.printf("Begin in 1-bit mode; pins: CLK=%d, CMD=%d, D0=%d\n", SD_MMC.getClkPin(), SD_MMC.getCmdPin(), SD_MMC.getD0Pin());
+    } else {
+        Serial.printf("Begin in 4-bit mode; pins: CLK=%d, CMD=%d, D0=%d, D1=%d, D2=%d, D3=%d\n", SD_MMC.getClkPin(), SD_MMC.getCmdPin(), SD_MMC.getD0Pin(), SD_MMC.getD1Pin(), SD_MMC.getD2Pin(), SD_MMC.getD3Pin());
```

Comment:
The getters were removed from the PR for 3.0.0. Please remove this part.

Reply:
so there will be no way to read the pins? o.0